pkgs/by-name/vs/vscode-js-debug/package.nix

```diff
@@ -0,0 +1,88 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, jq
+, libsecret
+, pkg-config
+, nodePackages
+, runCommand
+, vscode-js-debug
+, nix-update-script
+}:
+
+buildNpmPackage rec {
+  pname = "vscode-js-debug";
+  version = "1.88.0";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "vscode-js-debug";
+    rev = "v${version}";
+    hash = "sha256-gmeuRUcdz/4+FtPOblNj5DX3otXNRjHJjhPcCRuWXAY=";
+  };
+
+  npmDepsHash = "sha256-M4h2p8GLVjBDla0ile1jKWF6wPSdgcumx2GKm9KGmlw=";
+
+  nativeBuildInputs = [ pkg-config nodePackages.node-gyp ];
+
+  buildInputs = [ libsecret ];
+
+  postPatch = ''
+    ${jq}/bin/jq '
+      .scripts.postinstall |= empty |             # tries to install playwright, not necessary for build
+      .scripts.build |= "gulp dapDebugServer" |   # there is no build script defined
+      .bin |= "./dist/src/dapDebugServer.js"      # there is no bin output defined
+    ' ${src}/package.json > package.json
+  '';
+
+  makeCacheWritable = true;
+
+  npmInstallFlags = [ "--include=dev" ];
+
+  preBuild = ''
+    export PATH="node_modules/.bin:$PATH"
+  '';
+
+  passthru.updateScript = nix-update-script {
+    extraArgs = [ "--version-regex" "v((?!\d{4}\.\d\.\d{3}).*)" ];
+  };
+
+  passthru.tests.test = runCommand "${pname}-test"
+    {
+      nativeBuildInputs = [ vscode-js-debug ];
+      meta.timeout = 60;
+    } ''
+    if output=$(js-debug --help 2>&1); then
+      if grep -Fw -- "Usage: dapDebugServer.js [port|socket path=8123] [host=localhost]" - <<< "$output"; then
+        touch $out
+      else
+        echo "Expected help output was not found!" >&2
+        echo "The output was:" >&2
+        echo "$output" >&2
+        exit 1
+      fi
+    else
+      echo "js-debug returned a non-zero exit code." >&2
+      echo "$output" >&2
+      exit 1
```

Comment:
I am like 50% sure that this already gets printed in the build log

Reply:
I confirm:
```
error: builder for '/nix/store/7iqzy2j3w9zw8r0ckz0ax9x82xnjcyk6-vscode-js-debug-test.drv' failed with exit code 3
```

The whole `if/else` block can be deleted.